libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/metrics/NavigationMetricsWrapper.java

```diff
@@ -180,7 +180,7 @@ static void feedbackEvent(SessionState sessionState, MetricsRouteProgress routeP
     MapboxTelemetry.getInstance().flushEventsQueueImmediately(false);
   }
 
-  static void turnstileEvent() {
+  public static void turnstileEvent() {
```

Comment:
This shouldn't be public as someone could just call this static method and send many turnstile events.